package.json

```diff
@@ -47,16 +47,16 @@
     "husky": "^8.0.3",
     "jasmine-core": "~4.6.0",
     "jasmine-spec-reporter": "~7.0.0",
-    "karma": "~6.4.2",
-    "karma-chrome-launcher": "~3.2.0",
-    "karma-coverage-istanbul-reporter": "~3.0.3",
-    "karma-jasmine": "~5.1.0",
-    "karma-jasmine-html-reporter": "^2.0.0",
+    "karma": "~6.4.1",
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage-istanbul-reporter": "^3.0.2",
+    "karma-jasmine": "~4.0.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
     "lint-staged": "^13.2.2",
     "ng-packagr": "^16.0.0",
     "protractor": "~7.0.0",
     "ts-node": "~10.9.1",
-    "typescript": "^5.0.4"
+    "typescript": "~4.6.4",
```

Comment:
This line should not have a trailing comma as it breaks the json format and npm is unable to use the file